index.html

```diff
@@ -63,28 +63,37 @@ <h2 id="project_tagline"></h2>
 <pre>$ java -jar publish.war -h
 You passed help flag.
 usage: java -jar
```

Comment:
Somewhere in the documentation we need to describe how the routing key is created.